README.md

```diff
@@ -12,15 +12,14 @@
 2. Setup your project config in `.eslintrc.js`:
 
 ```js
-// This enables ESLint to use dependencies of this config
-// (see https://github.com/eslint/eslint/issues/3458)
-require('eslint-config-molindo/setupPlugins');
-
 module.exports = {
   // Add configs based on your needs
   extends: [
-    'molindo/javascript', // Or `molindo/typescript`
+    'molindo/typescript', // Or `molindo/javascript`
+    
     'molindo/react', // Optional
+    'molindo/css-modules', // Optional
+    'molindo/tailwind', // Optional
```

Comment:
Is it ok if this is included, even though it's currently not used in or planned for Molindo projects?

I'm keeping `eslint-config-molindo` maintained also for other projects, that's the reason why it already supported TypeScript for example.

Is it ok if there's a minimal balance that this library might provide something not directly useful to Molindo, but in exchange also benefits from improvements from other projects since still the majority overlaps? Currently Tailwind would be the only preset falling in this category and I don't have anything else in mind currently. Also it doesn't affect the presets that are in use for Molindo in a negative way.